samples/deployment/test-network/installFPC.sh

```diff
@@ -23,8 +23,12 @@ PEERS=("peer0.org1.example.com" "peer0.org2.example.com")
 ERCC_EP="OutOf(2, 'Org1MSP.peer', 'Org2MSP.peer')"
 ECC_EP="OutOf(2, 'Org1MSP.peer', 'Org2MSP.peer')"
 
-CC_ID="echo"
-CC_VER="$(cat ${FPC_PATH}/samples/chaincode/${CC_ID}/_build/lib/mrenclave)"
+if [[ -z "${TEST_CC_ID}" ]]; then
+  echo "TEST_CC_ID not set - using echo chaincode"
+  TEST_CC_ID=echo
+#  exit 1
+fi
```

Comment:
To make this install script more flexible I think we should make a bit more flexibel.

I think the default chaincode used here can still be `echo` and making this echo output makes sense.

So what would be the best way to install a custom chaincode? In order to do that we need the CC_PATH and CC_ID

CC_ID= read from env; if not present use `echo`
CC_PATH= read from env; if not present use `${FPC_PATH}/samples/chaincode/echo/`

Then CC_VER becomes
`CC_VER="$(cat ${CC_PATH}/_build/lib/mrenclave)"`

Reply:
This will be covered in #605